src/snowflake/connector/cursor.py

```diff
@@ -117,6 +117,19 @@ class ResultMetadata(NamedTuple):
     scale: int
     is_nullable: bool
 
+    @classmethod
+    def from_column(cls, col):
+        type_value = FIELD_NAME_TO_ID[col["type"].upper()]
+        return cls(
+            col["name"],
+            type_value,
```

Comment:
It's never re-used, lets inline it
```suggestion
        return cls(
            col["name"],
            FIELD_NAME_TO_ID[col["type"].upper()],
```